app/assets/stylesheets/components/_navbar.scss

```diff
@@ -9,8 +9,47 @@
   font-weight: bold;
   .content {
     position: absolute;
-    left: 50%;
-    top: 50%;
+    left: 35%;
+    top: 35%;
```

Comment:
Using relative values for vertical positioning is tricky in my experience. Please turn into to an absolute value. No pb using relative values for horizontal positioning though.

Reply:
Done